tuna/rocmlir/rocmlir_tables.py

```diff
@@ -188,22 +189,83 @@ def __repr__(self) -> str:
 
   def config_string(self):
     """Return config as a flag/value string suitable for tuningRunner.py."""
-    string = "conv "  # +++pf:  of course generalise for gemm
-    for field, value in self.to_dict().items():
-      flag = self.options[field]
-      if flag:
+    string = "conv"
+    if self.data_type == 'f16':
+      string += 'fp16'
+    elif self.data_type == 'i8':
+      string += 'int8'
+    elif self.data_type == 'bf16':
+      string += 'bfp16'
+    string += " "
+
+    # In options order for canonicalisation, kind of.
```

Comment:
Always writing config options in the same order, because the .tsv lookup in the performance runs does a simple string match to find the corresponding perf-config.  The order is fixed by using the list of option-field pairs.